docs/en/user_guides/train_test.md

```diff
@@ -0,0 +1,350 @@
+```
+
+Next, we will specify the details of different metrics one by one.
+
+### **FID** and **TransFID**
+
+Fréchet Inception Distance is a measure of similarity between two datasets of images. It was shown to correlate well with the human judgment of visual quality and is most often used to evaluate the quality of samples of Generative Adversarial Networks. FID is calculated by computing the Fréchet distance between two Gaussians fitted to feature representations of the Inception network.
+
+In `MMGeneration`, we provide two versions for FID calculation. One is the commonly used PyTorch version and the other one is used in StyleGAN paper. Meanwhile, we have compared the difference between these two implementations in the StyleGAN2-FFHQ1024 model (the details can be found [here](https://github.com/open-mmlab/mmgeneration/blob/master/configs/styleganv2/README.md)). Fortunately, there is a marginal difference in the final results. Thus, we recommend users adopt the more convenient PyTorch version.
+
+**About PyTorch version and Tero's version:** The commonly used PyTorch version adopts the modified InceptionV3 network to extract features for real and fake images. However, Tero's FID requires a [script module](https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/metrics/inception-2015-12-05.pt) for Tensorflow InceptionV3. Note that applying this script module needs `PyTorch >= 1.6.0`.
+
+**About extracting real inception data:** For the users' convenience, the real features will be automatically extracted at test time and saved locally, and the stored features will be automatically read at the next test. Specifically, we will calculate a hash value based on the parameters used to calculate the real features, and use the hash value to mark the feature file, and when testing, if the `inception_pkl` is not set, we will look for the feature in `MMGEN_CACHE_DIR` (~/.cache/openmmlab/mmgen/). If cached inception pkl is not found, then extracting will be performed.
+
+To use the FID metric, you should add the metric in a config file like this:
+
+```python
+metrics = [
+    dict(
+        type='FrechetInceptionDistance',
+        prefix='FID-Full-50k',
+        fake_nums=50000,
+        inception_style='StyleGAN',
+        sample_model='ema')
+]
+```
+
+If you work on an new machine, then you can copy the `pkl` files in `MMGEN_CACHE_DIR` and copy them to new machine and set `inception_pkl` field.
+
+```python
+metrics = [
+    dict(
+        type='FrechetInceptionDistance',
+        prefix='FID-Full-50k',
+        fake_nums=50000,
+        inception_style='StyleGAN',
+        inception_pkl=
+        'work_dirs/inception_pkl/inception_state-capture_mean_cov-full-33ad4546f8c9152e4b3bdb1b0c08dbaf.pkl',  # copied from old machine
+        sample_model='ema')
+]
+```
+
+`TransFID` has same usage as `FID`, but it's designed for translation models like `pix2pix` and `cyclegan`, which is designed for our evaluator. You can refer
```

Comment:
pix2pix or Pix2Pix
cyclegan or CycleGAN

We should unify the writing style of these two models.